drawio_desktop/src/mermaid-plugin.js

```diff
@@ -147,23 +233,160 @@ var DialogMermaid = function (editorUi, shape) {
 };
 
 Draw.loadPlugin(function (ui) {
+
+  // Build mermaid settings : by least order
+  // - mermaid_plugin_defaults : this plugin defaults
+  // - EditorUi.defaultMermaidConfig : drawio defaults mermaid
+  // - Editor.config.defaultMermaidConfig : drawio config (from PreConfig and local configuration)
+
+  let mermaid_settings = {};
+  mermaid_settings = mergeDeep(mermaid_settings, mermaid_plugin_defaults);
+  mermaid_settings = mergeDeep(mermaid_settings, window.EditorUi.defaultMermaidConfig);
+  mermaid_settings = mergeDeep(mermaid_settings, window.Editor.config.defaultMermaidConfig);
```

Comment:
I had problems here with the latest drawio-desktop version.
```suggestion
  mermaid_settings = mergeDeep(mermaid_settings, mermaid_plugin_defaults);
  try {
    mermaid_settings = mergeDeep(mermaid_settings, window.EditorUi.defaultMermaidConfig);
  } catch (e) {
    if (!e instanceof TypeError) {
      throw e;
    }
  }
  try {
    mermaid_settings = mergeDeep(mermaid_settings, window.Editor.config.defaultMermaidConfig);
  } catch (e) {
    if (!e instanceof TypeError) {
      throw e;
    }
  }
```